venues/reproducibility-challenge.github.io/Reproducibility_Challenge/NeurIPS/2019/python/comment_notification.py

```diff
@@ -0,0 +1,102 @@
+
+import argparse
+import openreview
+from openreview import tools
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print("Connected to "+client.baseurl)
+conference_id = 'NeurIPS.cc/2019/Reproducibility_Challenge'
+
+# for each submission, create group (so sub invites can be published)
+# create invitation for notification frequency tags
+# create invite for comment
+notes = tools.iterget_notes(client,invitation=conference_id+"/-/NeurIPS_Submission")
+for note in notes:
+    # need paper group to publish sub-groups
+    paper_group = client.post_group(openreview.Group(
+        id='{conference_id}/NeurIPS{number}'.format(conference_id=conference_id, number=note.number),
+        signatures=[conference_id], signatories=[conference_id],
+        readers=[conference_id], writers=[conference_id]))
+
+    notify_inv = openreview.Invitation(
+        id='{}/NeurIPS{}/-/Comment_Notification'.format(conference_id, note.number),
+        readers=['everyone'],
+        invitees=['~'],
+        writers=[conference_id],
+        signatures=[conference_id],
+        details= {'writable':True},
+        multiReply= False,
+        reply={
+            'forum': note.forum,
```

Comment:
invitation: 'NeurIPS.cc/2019/Reproducibility_Challenge/-/NeurIPS_Submission', remove forum and replyto.